src/meta/processors/indexMan/CreateEdgeIndexProcessor.cpp

```diff
@@ -16,12 +16,20 @@ void CreateEdgeIndexProcessor::process(const cpp2::CreateEdgeIndexReq& req) {
     auto &edgeName = req.get_edge_name();
     auto &fieldNames = req.get_fields();
     if (fieldNames.empty()) {
-        LOG(ERROR) << "Edge's Field should not empty";
+        LOG(ERROR) << "Edge Index's field should not empty";
```

Comment:
The index field of an edge type should not be empty.

Reply:
Good Point ~~